lib/kanal/plugins/user_system/user_system_plugin.rb

```diff
@@ -43,7 +42,7 @@ def setup(core)
           setup_user_storage core
           setup_user_state core
 
-          enable_telegram(core) if @auto_create.telegram_enabled
+          AutoCreator.new.enable_telegram(core) if @auto_create.telegram_enabled
```

Comment:
No need to use instancing in this case. Simply calling module method will suffice